docs/DEV_PREVIEW.md

```
@@ -0,0 +1,166 @@
+
+# With mock API data
+python scripts/render_plugin.py --plugin football-scoreboard \
+  --mock-data /tmp/mock_scores.json \
+  --output /tmp/football.png
+```
+
+### Using with Claude Code / AI
+
+Claude can run the render script, then read the output PNG (Claude is multimodal and can see images). This enables a visual feedback loop:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)